site/zenodo_rdm/migrator/transform/entries/records/metadata.py

```diff
@@ -287,8 +285,7 @@ def transform(cls, entry):
             ),
             "references": cls._references(entry.get("references")),
             "dates": cls._dates(entry.get("dates")),
-            # TODO: fix location entry
-            # "locations": cls._locations(entry.get("locations")),
+            "locations": cls._locations(entry.get("locations")),
```

Comment:
Needs to be added to drafts too (and tests also adapted). Either in this PR or in https://github.com/zenodo/zenodo-rdm/pull/286 

I think we can merge this and tackle changes in mine? WDYT

Reply:
Added!